### DISCUSSION TITLE
Pairing the Linkind motion sensor ZS110040078

### DISCUSSION THREAD

**Discussion by evilpie** [Koenkk/zigbee2mqtt#8338] [Devices]
I bought the "Linkind Smart Home Security System Starter Kit" which includes a motion sensor with the model number `ZS110040078`, which seems to be different than the already supported `ZS1100400-IN-V1A02`. Unlike the included door sensors, which have a reset hole, I can't figure out how to pair the motion sensor.

Has anyone got this working before? @GSzabados maybe or someone else actively working on https://github.com/Koenkk/zigbee2mqtt/issues/6242.

**Comment by GSzabados** [comment]
Hi @evilpie,

I have just paired my sensor yesterday evening and paired with the `ZS1100400-IN-V1A02` as expected.

When you do pair your sensor does it show up as not supported?

Can you share a screenshot from Z2M what it shows?

Can you share some images from the sensor as well? I am curious how much is it different from what I have.

And last, which starter kit have you got the sensor from? With the keypad or with the remote?

Otherwise the procedure would be the standard one to support a currently unsupported device. You can find it in the documentation, but I can give a hand with it. You will need to create and external_converter for the device with the right "fingerprint". Suspecting that the device is the same as the other ones, then I would use the code from the old one, then I would add the `ZS110040078` to that. If all features are working an no missing clusters in the debug logging, then it will be good to create a PR by adding just the new name. It is not so complicated.

Best Regards,

Gábor

**Comment by GSzabados** [comment]
I forgot to mention to start pairing the device, it should have a button under the battery cover. The manual should have details how you need to press but a search for `ZS110040078` brings up the FCC filing and the manual as well, if you miss that for some reason.

**Comment by evilpie** [comment]
Oh wow. I am feeling a bit stupid right now. I was looking for some kind of little hole to put the pin tool in and didn't realize that the little plastic tab right next the `+` can be pushed down! Thank you so much, it's working now.

Aside: I think I also noticed that the sensor doesn't seem to report data when open.

**Reply by GSzabados** [reply]
> Aside: I think I also noticed that the sensor doesn't seem to report data when open.

Do you mean the battery cover open, or it is not fixed inside the holder? The second case is more reasonable as it has been tampered.

Otherwise, have you turned off pairing in Z2M when you saw this?

**Reply by evilpie** [reply]
> Do you mean the battery cover open, or it is not fixed inside the holder? The second case is more reasonable as it has been tampered.

The cover was open and the holder wasn't attached. I guess it doesn't really matter, the sensor works in normal operation.

**Comment by stefangries** [comment]
I paired my `ZS110040078 `as well, and it was recognized as `ZS1100400-IN-V1A02`, too.
![image](https://user-images.githubusercontent.com/10342151/132393327-a2b20c69-6d97-423e-a3ad-f6f96c1d4e29.png)

But: I don't get any values. Any idea, how to fix this?
![image](https://user-images.githubusercontent.com/10342151/132393409-a26f48a0-3a61-4999-8f82-2f8cdf377c0b.png)
Repeated pairing, even near the coordinator, did not help.

I noticed, that there aren't any bindings. Do you have one?
![image](https://user-images.githubusercontent.com/10342151/132394052-8bab551c-8c24-4fac-a129-db1f23dd8107.png)

**Reply by evilpie** [reply]
Sorry, I don't have much experience with z2m yet either. I would suggest going to Logs and changing the Log level to debug and reconnecting the device. Maybe you can see some kind of error.

**Reply by stefangries** [reply]
I have already done that, but couldn't find anything suspicious.  But if you don't know, what you are looking for, it's difficult to find.

**Reply by GSzabados** [reply]
Put back the battery cover.

**Reply by stefangries** [reply]
Already done :-(
Btw, the LED of the device keeps flashing, although it already appeared in zigbee2mqtt.

**Reply by GSzabados** [reply]
Force delete the device from Z2M. Remove battery, put battery back, reset the device with the pinhole, and start pairing with Z2M. Wait until it finishes the interview process.

**Reply by stefangries** [reply]
The ZS110040078 doesn't have a pinhole, but a button that only can be reached when battery door is open.

**Reply by GSzabados** [reply]
Sorry, I mixed with the keypad and the ?door sensor?, yes that button I meant.

**Reply by stefangries** [reply]
What I did: remove battery, force delete from z2m, insert battery, push button to pair, enable z2m join.
The device joined, but problem is as before. No values.

Are you sure, that we are talking about the same device here? My doesn't have a door sensor but only a pair button. So it shouldn't make any difference if the battery door is open or closed. Maybe the ZS110040078 is simply not supported?
And: Could you please have a look at the bindings I mentioned before? Maybe there is something missing.
![image](https://user-images.githubusercontent.com/10342151/132487466-15acfd36-0b2c-4dee-801e-d251c3f2977b.png)

**Reply by GSzabados** [reply]
I meant, the door (open/close) sensor has a pinhole.

For some reason when finished pairing the battery cover has to be on to start working.

If you put back the battery cover, and the holder as well. Try to leave it without any motion for 5 minutes. The move your hand in front of it to start motion detection and hit reconfigure. (The yellow/orange icon with the circular arrows.)

**Reply by stefangries** [reply]
Did it exactly as you said, but getting this error when hitting reconfigure:
![image](https://user-images.githubusercontent.com/10342151/132490718-ca30798f-157f-45a7-ab20-4d373b2fc995.png)

**Reply by GSzabados** [reply]
> Did it exactly as you said, but getting this error when hitting reconfigure:

It is a sleepy end device. That's why it is important to be awake when trying to reconfigure.

**Reply by GSzabados** [reply]
> but getting this error

You can trigger a wake up by removing or putting back the holder bracket of the device. That would trigger a tamper alert message and would wake up the device.

I am not sure that the bracket is required for the device or not to operate normally.

I have one more motion sensor at home. I might going to try it this afternoon.

**Reply by stefangries** [reply]
Thanks. In my case, putting off and on the bracket does't makes it reconfigurable.

**Reply by GSzabados** [reply]
By the way, so you get any messages from it? Is the last seen updates at all?

**Reply by stefangries** [reply]
I don't think I get anything from the device. That's why I asked for the binding, and if it is correct, that the device keeps blinking although it already appeared in z2m.

**Reply by GSzabados** [reply]
There shouldn't be anything else required to set up. It should do it by itself.

**Reply by stefangries** [reply]
While I was trying to open the housing of my sensor, I destroyed mine. I wanted to see if the door sensor was triggered correctly. So you don't have to invest any more time for me.

**Comment by TechnicallyBob202** [comment]
I'm having the same problem.  Interview fails with the following log:  `Failed to configure '0x04cd15fffe0b6d12', attempt 1 (Error: Bind 0x04cd15fffe0b6d12/1 genPowerCfg from '0x00212effff0736be/1' failed (waiting for response TIMEOUT) at DeconzAdapter.bind (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/src/adapter/deconz/adapter/deconzAdapter.ts:757:19) at Endpoint.bind (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/src/controller/model/endpoint.ts:500:13) at Object.bind (/opt/zigbee2mqtt/node_modules/zigbee-herdsman-converters/lib/reporting.js:33:9) at Object.configure (/opt/zigbee2mqtt/node_modules/zigbee-herdsman-converters/devices/linkind.js:115:13) at Configure.configure (/opt/zigbee2mqtt/lib/extension/configure.ts:119:13))`

**Reply by GSzabados** [reply]
@rlyons20, try this:

1. Move the device closer to the zigbee coordinator. It can be a few centimeters if possible...
2. Delete the device from Z2M.
3. Restart Z2M.
4. Start permit to join.
5. Start pairing the device.
6. Make sure that the device doesn't go into sleeping state, so wave your hand in front of the sensor, time to time, or pop in and out of the wall mount bracket, close the battery lid, etc. Anything what makes the device to stay awake.

At this stage the device should show up in Z2M and should pass the interview and configuration correctly.

**Reply by TechnicallyBob202** [reply]
I did all of this...  I will try again...

**Reply by GSzabados** [reply]
The order is important. Especially deleting and restarting Z2M. 

The timeout usually happens when the devices goes to sleep during the interview or configure process.

**Reply by TechnicallyBob202** [reply]
I tried that exact order, and same behavior

**Reply by GSzabados** [reply]
Does it say at the end interview failed? If not, then just try to hit the reconfigure button.
